cpp/BUILD

```diff
@@ -53,7 +53,13 @@ stamped_plugin_xml(
 
 optional_plugin_xml(
     name = "optional_cidr",
-    module = ["com.intellij.modules.cidr.lang"],
```

Comment:
`com.intellij.modules.cidr.lang` -> `com.intellij.cidr.lang` should be ok for old versions too

Reply:
because of 
```
    {
      "name": "com.intellij.cidr.lang",
      "kind": "plugin",
      "classPath": [
        "plugins/c-plugin/lib/c-plugin.jar"
      ]
    },
``` 

in `product-info.json`

Reply:
Yes, looks like tests are also passing using just `com.intellij.cidr.lang`. I think I would still like to keep the `select_for_version` since it might useful in future.